copier.yml

```diff
@@ -109,16 +109,19 @@ _tasks:
   - "{% if not documentation %}rm -rf docs .readthedocs.yml{% endif %}"
   # Run 'uv sync' to generate lockfile
   - "{% if run_uv_sync %}uv sync{% endif %}"
-  # Initial commit
+  # Initial commit (1)
   - "{% if initial_commit %}git init{% endif %}"
   - "{% if initial_commit %}git add .{% endif %}"
+  # Setup pre-commit
+  - "{% if setup_pre_commit %}pre-commit install{% endif %}"
+  - "{% if setup_pre_commit %}pre-commit run -a || true{% endif %}"
+  # Initial commit (2) (assuming that pre-commit will not raise an error on the second run)
```

Comment:
I think that's why I didn't do it before: because this can easily fail and that would make the whole thing fail over and interrupt the next steps. I feel like the following steps are more important to complete that having a clean pre-commit at this point

Reply:
Either way, it is still a hassle if pre-commit fails. But the advantage is, though in either case, copier will delete the local files, without this change the user would additionally still have a broken repository on github.

Reply:
> in either case copier will delete the local files

I don't think it does currently, does it? We don't call `pre-commit run -a` at all right now... If pre-commit doesn't pass, the user has all the files and the power to power to fix the problem. I wouldn't go as far as calling this a "broken" repository though 😄

Reply:
I don't remember much about the behavior of Copier, but if that's the case, what about running (only) `pre-commit run -a || true` before initial commit and keep `pre-commit install` as it is?